pytorch/training/buildspec-1-13-neuronx.yml

```diff
@@ -0,0 +1,43 @@
+account_id: &ACCOUNT_ID <set-$ACCOUNT_ID-in-environment>
```

Comment:
Is this a duplicate of the original `buildspec`?

Reply:
It's based on the previous releases' buildspec.